webrtc/RTCPeerConnection-addTrack.https.html

```diff
@@ -64,6 +64,62 @@
               transceiver be the result.
           4.  Add transceiver to connection's set of transceivers.
    */
+  function addTrackFromGetUserMedia(t) {
+    const pc = new RTCPeerConnection();
+    t.add_cleanup(() => pc.close());
+
+    return navigator.mediaDevices.getUserMedia({ audio: true })
+    .then(mediaStream => {
+      const track = mediaStream.getTracks()[0];
+      t.add_cleanup(() => track.stop());
+      const sender = pc.addTrack(track, mediaStream);
+      return {pc, track, sender};
+    });
+  }
+
+  promise_test(t => {
```

Comment:
Here and elsewhere: Can you update to use promise_test(async t => { ... while you're at it? :) at least for the tests you're updating.

Reply:
done locally.